<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-02T17:50:29 IST" name="Default suite" finished-at="2022-09-02T17:50:54 IST" duration-ms="25494">
    <groups>
    </groups>
    <test started-at="2022-09-02T17:50:29 IST" name="Default test" finished-at="2022-09-02T17:50:54 IST" duration-ms="25494">
      <class name="module1.kite_login1_test">
        <test-method is-config="true" signature="openbrowser()[pri:0, instance:module1.kite_login1_test@18d87d80]" started-at="2022-09-02T17:50:29 IST" name="openbrowser" finished-at="2022-09-02T17:50:36 IST" duration-ms="6804" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openbrowser -->
        <test-method signature="TC1()[pri:0, instance:module1.kite_login1_test@18d87d80]" started-at="2022-09-02T17:50:36 IST" name="TC1" finished-at="2022-09-02T17:50:37 IST" duration-ms="1748" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC1 -->
        <test-method is-config="true" signature="closeurl(org.testng.ITestResult)[pri:0, instance:module1.kite_login1_test@18d87d80]" started-at="2022-09-02T17:50:37 IST" name="closeurl" finished-at="2022-09-02T17:50:42 IST" duration-ms="4573" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=TC1 status=SUCCESS method=kite_login1_test.TC1()[pri:0, instance:module1.kite_login1_test@18d87d80] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeurl -->
        <test-method is-config="true" signature="openbrowser()[pri:0, instance:module1.kite_login1_test@18d87d80]" started-at="2022-09-02T17:50:42 IST" name="openbrowser" finished-at="2022-09-02T17:50:47 IST" duration-ms="5026" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openbrowser -->
        <test-method signature="TC2()[pri:0, instance:module1.kite_login1_test@18d87d80]" started-at="2022-09-02T17:50:47 IST" name="TC2" finished-at="2022-09-02T17:50:50 IST" duration-ms="2636" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[failed=error msg is not same expected [ Invalid username or password.] but found [Invalid username or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: failed=error msg is not same expected [ Invalid username or password.] but found [Invalid username or password.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at module1.kite_login1_test.TC2(kite_login1_test.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC2 -->
        <test-method is-config="true" signature="closeurl(org.testng.ITestResult)[pri:0, instance:module1.kite_login1_test@18d87d80]" started-at="2022-09-02T17:50:50 IST" name="closeurl" finished-at="2022-09-02T17:50:54 IST" duration-ms="4637" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=TC2 status=FAILURE method=kite_login1_test.TC2()[pri:0, instance:module1.kite_login1_test@18d87d80] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeurl -->
      </class> <!-- module1.kite_login1_test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
